src/pybind/mgr/cephadm/serve.py

```diff
@@ -410,21 +410,22 @@ def _check_for_strays(self) -> None:
                     daemon_id = s.get('id')
                     assert daemon_id
                     name = '%s.%s' % (s.get('type'), daemon_id)
-                    if s.get('type') in ['rbd-mirror', 'cephfs-mirror', 'rgw']:
+                    if s.get('type') in ['rbd-mirror', 'cephfs-mirror', 'rgw', 'rgw-nfs']:
                         metadata = self.mgr.get_metadata(
```

Comment:
needs a mock in the unit test:
```E       AttributeError: 'CephadmOrchestrator' object has no attribute '_ceph_get_metadata'```

Reply:
let me know if 90dcb68b196b2b6227f79c8a73b9206fc963532c is the correct way to mock this